vaadin-core-internal/pom.xml

```diff
@@ -209,6 +209,11 @@
             <artifactId>vaadin-menu-bar-flow</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.vaadin.hilla</groupId>
+            <artifactId>hilla</artifactId>
```

Comment:
Would it be better to add `hilla` to `vaadin` until at least we don't have a split for core and all react components?

Reply:
Also, the idea to add `hilla` to `vaadin-spring-boot-starter` looks not that bad to me now. As Jakarta EE and Quarkus users can just not add `vaadin-spring-boot-starter` to their pom.xml and they won't have Spring and Hilla then.

Reply:
This would also make it pretty easy to exclude hilla by not using `vaadin-spring-boot-starter` because it is only a wrapper for `vaadin-spring`, `vaadin-core` and `hilla`. So the flow developer only has to add `vaadin-spring` and `vaadin-core` if they don't wanna deal with exclusions.

Reply:
This would also give easy option to divide it in two platform modules like `vaadin-spring-boot-starter` and `vaadin-hilla-spring-boot-starter`.

Reply:
The idea is to use one dependency `vaadin-spring-boot-starter` and avoid "hilla" artifacts, so having a second one with the hilla in the middle is approx. the same as have old `hilla-spring-boot-starter`.